src/main/java/org/tron/core/db/Manager.java

```diff
@@ -317,7 +317,7 @@ public AccountContractIndexStore getAccountContractIndexStore() {
             this.rePush(tx);
           } catch (InterruptedException ex) {
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Either re-interrupt this method or rethrow the "InterruptedException". ([squid:S2142](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2142))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1324&issues=AWVbUex9ZiUEgfxFv6nB&open=AWVbUex9ZiUEgfxFv6nB)

[ISSUE_KEY]: AWVbUex9ZiUEgfxFv6nB